tests/unit/backend/models/form_fields.schema.spec.js

```diff
@@ -0,0 +1,126 @@
+const { merge } = require('lodash')
+const mongoose = require('mongoose')
+
+const dbHandler = require('../helpers/db-handler')
+const Form = require('../../../../dist/backend/app/models/form.server.model').default(
+  mongoose,
+)
+
+describe('Form Field Schema', () => {
+  let collections
+
+  beforeAll(async () => await dbHandler.connect())
+  beforeEach(async () => {
+    collections = await dbHandler.preloadCollections({ saveForm: false })
+  })
+  afterEach(async () => await dbHandler.clearDatabase())
+  afterAll(async () => await dbHandler.closeDatabase())
+
+  describe('Methods', () => {
+    describe('getQuestion', () => {
+      it('should return field title when field type is not a table field', async () => {
+        const {
+          BasicFieldType,
+        } = require('../../../../dist/backend/types/field/fieldTypes')
+
+        // Arrange
+        // Get all field types
+        const fieldTypes = Object.values(BasicFieldType)
+
+        // Asserts
+        fieldTypes.forEach(async (type) => {
+          // Skip table field.
+          if (type === 'table') return
+          const fieldTitle = `test ${type} field title`
+          const field = await createAndReturnFormField({
+            fieldType: type,
+            title: fieldTitle,
+          })
+
+          // Assert
+          expect(field.getQuestion()).toEqual(fieldTitle)
+        })
+      })
+
+      it('should return table title concatenated with all column titles when field type is a table field', async () => {
+        // Arrange
+        const tableFieldParams = {
+          title: 'testTableTitle',
+          minimumRows: 1,
+          columns: [
+            {
+              title: 'Test Column Title 1',
+              required: true,
+              columnType: 'textfield',
+            },
+            {
+              title: 'Test Column Title 2',
+              required: true,
+              columnType: 'dropdown',
+            },
+          ],
+          fieldType: 'table',
+        }
+
+        // Act
+        const tableField = await createAndReturnFormField(tableFieldParams)
+
+        // Assert
+        const expectedQuestionString = `${
+          tableFieldParams.title
+        } (${tableFieldParams.columns.map((col) => col.title).join(', ')})`
+        expect(tableField.getQuestion()).toEqual(expectedQuestionString)
+      })
+    })
+  })
+
+  const createAndReturnFormField = async (formFieldParams, formType) => {
```

Comment:
the need for this is telling me that we're missing a factory method to create form field objects - we should eventually put one on `FieldSchema`.

Reply:
Yes, since `FormFields` are not registered as a model, we can't create them directly. Wouldn't creating a method still require the model to be registered?